src/components/common/EffectiveDate.vue

```diff
@@ -0,0 +1,198 @@
+<template>
+  <v-card flat id="effective-date">
+      <v-row>
+        <v-col cols="12" sm="3">
+          <label class="title-label">Effective Date of Continuation Out</label>
+        </v-col>
+        <v-col cols="12" sm="9">
+          <v-menu
```

Comment:
Could you use the shared date picker here?

Reply:
Example: https://github.com/bcgov/business-create-ui/blob/a9fb48d2efece324827d8333bddf84ab1ad539d7/src/components/Registration/StartDate.vue#L9

Reply:
Looks like you copied other code from this project, so I'm OK if you leave this as-is.

Reply:
Yes! You could say this a spin-off version of the CODDate component with improved design (grid system).